pkgs/development/libraries/kerberos/krb5.nix

```diff
@@ -10,58 +18,50 @@
 , python3
 
 # Extra Arguments
-, type ? ""
+, withLdap ? false, openldap
+, withLibedit ? true, libedit
```

Comment:
Can you provide us with a rationale behind the inclusion of `byacc` and `libedit`?

Reply:
Ultimately, the reason we used to have `libOnly` was to avoid `bison` and `libedit` dependencies. Using those makes it much harder to break the cycle `fetchurl -> curl -> krb5 -> fetchurl`. I was able to solve that by...

  1. `libedit` is actually an optional dependency if you just disable it with a `configure` flag, so I made it optional and had `fetchurl` disable it.
  2. `bison` has a large build-time closure, but `byacc` has a tiny one. Thankfully, `krb5` says it can use either one, so I switched to `byacc` to make it very easy to break the cycle.

Reply:
I've updated the commit message to include this detail and explain the situation better.